src/observers/strategies/cpu_contention.hpp

```diff
@@ -50,26 +50,22 @@ class CpuContentionStrategy : public RevocationStrategy {
       const SerenityConfig& _config,
       const lambda::function<usage::GetterFunction>& _cpuUsageGetFunction)
       : RevocationStrategy(Tag(QOS_CONTROLLER, "CpuContentionStrategy")),
-        cpuUsageGetFunction(_cpuUsageGetFunction),
-        cooldownCounter(None()),
-        estimatorDisabled(false) {
+        getCpuUsage(_cpuUsageGetFunction) {
     SerenityConfig config = CpuContentionStrategyConfig(_config);
-    this->cfgCooldownTime = config.getU64(strategy::CONTENTION_COOLDOWN);
-    this->cfgDefaultSeverity = config.getD(strategy::DEFAULT_CPU_SEVERITY);
   }
 
-  RevocationStrategyFunction decide;
+  Try<QoSCorrections> decide(ExecutorAgeFilter* ageFilter,
```

Comment:
If anyone would like to implement ```decide``` function, one would need to check this alias.
I think it is more obscuring, because developer needs to constantly translate ```RevocationStrategyFunction``` to it's real signature.